eng/Version.Details.xml

```diff
@@ -105,5 +105,11 @@
       <Sha>9e7fbcab4e5275f63c0cd37553ba426de9194309</Sha>
       <SourceBuild RepoName="xliff-tasks" ManagedOnly="true" />
     </Dependency>
+    <!-- Necessary for source-build. This allows the package to be retrieved from previously-source-built artifacts
+         and flow in as dependencies of the packages produced by razor. -->
+    <Dependency Name="Microsoft.Extensions.ObjectPool" Version="6.0.0">
```

Comment:
Can this be a live version?  I am surprised the repo source-build legs don't encounter the same issue as the product build in that it is trying to load the reference package.

Reply:
Right, I understand that part.

I think I just answered my question in that the smoke-tests are what caught this.  We don't have anything that is running tests against the intermediate packages.

Reply:
Ideally this would be a live version in order to eliminate potential differences in the product source-build versus the source-build repo leg.

Reply:
Not sure that is possible with the tooling repos since they potentially go into multiple releases...IDK